.changeset/eix-73-group-a11y.md

```
@@ -0,0 +1,5 @@
+---
+'@siemens/ix': minor
+---
+
+Make `ix-group` keyboard- and screen-reader-accessible with separate header select and expand controls, disclosure ARIA (`aria-expanded` / `aria-controls`), Space/Enter, Escape to collapse, accessible names from the header text (expanded state from `aria-expanded` only), and screen-reader-compatible context-menu keyboard navigation.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)